src/libs/actions/connections/index.ts

```diff
@@ -403,6 +404,12 @@ function isConnectionUnverified(policy: OnyxEntry<Policy>, connectionName: Polic
     if (connectionName === CONST.POLICY.CONNECTIONS.NAME.NETSUITE) {
         return !(policy?.connections?.[CONST.POLICY.CONNECTIONS.NAME.NETSUITE]?.verified ?? true);
     }
+
+    // If the connection has no lastSync property, we'll consider it unverified
+    if (isEmptyObject(policy?.connections?.[connectionName]?.lastSync)) {
```

Comment:
Is this for all the connections?

Reply:
All non-netsuite connections yes. Those use the `lastSync` object.

Reply:
Got it.